src/panel/app_panel.js

```diff
@@ -254,12 +268,7 @@ export default class AppPanel {
     this.activePoiId = null;
     this.panels
       .filter(panel => panel !== panelToOpen)
-      .forEach(panel => {
-        if (panel === this.categoryPanel) {
-          fire('remove_category_markers');
-        }
-        panel.close();
-      });
+      .forEach(panel => { if (panel.close) { panel.close(); }});
```

Comment:
All panels have a `close` method, I don't think you need this test.

Reply:
indeed, I added it when I encountered "no function panel.close" errors, but the problem didn't come from here!